src/main/java/duke/Parser.java

```diff
@@ -0,0 +1,63 @@
+package duke;
+
+public class Parser {
+    enum modifyStatus {
+        MARK,
+        UNMARK
+    }
+
+    enum taskType {
+        EVENT,
+        DEADLINE,
+        TODO
+    }
+
+    /**
+     * Takes in commands from the user and does an action based on the command.
+     *
+     * @param nextLine input from the user.
+     * @param tasks list of task stored.
+     * @param storage storage for the tasks to be housed.
+     * @return lets the program know to stop
+     */
+    public static boolean parseCommands(String nextLine, TaskList tasks, Storage storage) {
+        String firstWord = nextLine.split(" ")[0];
+        try {
```

Comment:
I like the use of switch-case statements instead of if-else statements here, makes the code look cleaner!